db/migrate/20230503112500_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: 'users' }
+      t.string :title
+      t.string :text
+      t.integer :commrnts_counter, default: 0
+      t.integer :linkes_counter, default: 0
```

Comment:
- [ Required ] I think it would be better if we follow the column names on the `ERD` diagram, according to the ERD diagram these two columns should be `comments_counter` and `likes_counter`. Currently, it appears the names have been misspelled, please see the screenshot below and rectify this.

## ERD Diagram
![Screenshot from 2023-05-04 02-40-25](https://user-images.githubusercontent.com/16392046/236074113-7aa3cf8a-ca86-493e-8be1-ee86f65d9a30.png)